src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java

```diff
@@ -52,9 +52,14 @@ public CantGetEditAssetsException(String message, Throwable cause) {
         }
     }
 
-    public EditProtocol(ResourceIds resourceIds, ImageStore imageStore) {
+    public interface OnErrorListener {
```

Comment:
Yes, we can do that. But then EditProtocol will end up with having an instance of ViewCrawler instead of the onErrorListener interface, which gives EditProtocol too much power and looks bad.